src/application.py

```diff
@@ -35,8 +43,16 @@ def post_user(
     age: PositiveInt = None,
     country: CountryAlpha2 = None,
 ) -> User:
-    user_id = short_uuid4_generator()
+    """
+    Function that posts a user to the API, and adds it to the database. A check is done on whether a user with the same mail is present in the database. Although both age and country are optional, at least one of those must be provided, otherwise a ValidationError is thrown. It returns the user object.
+    :param username: string. Username. Only letters, more than two.
+    :param email: string. Email. Must be in the form aaa@bbb.ccc.
+    :param age: integer (optional). age in years.
+    :param country: string (optional). Country alpha-2 code. Two capitalized letters. check https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements if in doubt.
+    :return: User object.
+    """
```

Comment:
this docstring is very informative, nice!

Though it produces `/docs` page with docs that looks a bit scrambled:
![image](https://github.com/user-attachments/assets/2743d7b3-2bb9-440e-bd4d-1da1e3c2b10f)

do you think we could make it nicer? :)